src/XMLSecurityDSig.php

```diff
@@ -99,54 +120,91 @@ public function __construct($prefix='ds')
         $this->sigNode = $sigdoc->documentElement;
     }
 
-    private function resetXPathObj()
```

Comment:
Yes - and especially not using the "rearrange code" Feature - causing the worst mess.
Rearrange code is a nice feature - if everyone uses it.